packages/zk-eddsa-ticket-pcd/src/util/declarations/circomlibjs.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module "circomlibjs" {
```

Comment:
given that we use this in multiple places in our repo now, perhaps we can extract this into a package. if we do we can make that a different pr. cc @robknight

Reply:
makes sense, I can create an issue